docs/content/latest/api/ysql/extensions.md

```diff
@@ -265,171 +308,194 @@ If you encounter any issues when installing or using a particular extension, fil
 [PostGIS](https://postgis.net/) is a spatial database extender for PostgreSQL-compatible object-relational databases.
 The simplest way to set it up locally is to install it together with regular PostgreSQL.
 
+{{< note title="GiST index support" >}}
+
+YSQL does not yet support GiST indexes. This is tracked in [GitHub issue #1337](https://github.com/yugabyte/yugabyte-db/issues/1337).
+
+{{< /note >}}
+
 #### macOS
 
-- download and install [Postgres.app](https://postgresapp.com/)
-- install with Homebrew:
+There are two ways to install PostGIS on macOS:
+
+* Download and install [Postgres.app](https://postgresapp.com/)
+
+* Or, install with Homebrew:
 
     ```sh
     $ brew install postgres && brew install postgis
     ```
 
 #### Ubuntu
-Add the [PostgreSQL APT sources](https://www.postgresql.org/download/linux/ubuntu/). Then you can install:
+
+Add the [PostgreSQL APT sources](https://www.postgresql.org/download/linux/ubuntu/). Then, use `apt` to install:
 
 ```shell script
 sudo apt-get install postgresql-11 postgresql-11-postgis-3
 ```
 
 #### CentOS
-You can get the YUM repository from [PostgreSQL website](https://www.postgresql.org/download/linux/redhat/). Then you 
-can use `yum` or `dnf` to install:
+
+Get the YUM repository from the [PostgreSQL website](https://www.postgresql.org/download/linux/redhat/). Then, use `yum` or `dnf` to install:
+
 ```sh
 sudo yum install postgresql11-server postgis31_11
 ```
+
 {{< note title="Note" >}}
-If you have multiple PostgreSQL versions installed, make sure you're selecting the correct `pg_config`. On an Ubuntu 18.04 environment 
-with multiple PostgreSQL versions installed:
+
+If you have multiple PostgreSQL versions installed, make sure you're selecting the correct `pg_config`. On an Ubuntu 18.04 environment with multiple PostgreSQL versions installed:
+
 ```sh
 $ pg_config --version
 PostgreSQL 13.0 (Ubuntu 13.0-1.pgdg18.04+1)
+```
+
+```sh
 $ /usr/lib/postgresql/11/bin/pg_config --version
 PostgreSQL 11.9 (Ubuntu 11.9-1.pgdg18.04+1)
 ```
-So you need to make sure you're using the correct `pg_config`. In this case, `/usr/lib/postgresql/11/bin/pg_config`. 
+
+In this case, you should be using `/usr/lib/postgresql/11/bin/pg_config`. 
 
 On CentOS, the correct path is `/usr/pgsql-11/bin/pg_config`.
+
 {{< /note >}}
 
+#### Post-install
 
-Now, follow the instructions described above to copy the needed files into your YugabyteDB installation, and then create
-the extension.
+Now, follow the instructions described above to copy the needed files into your YugabyteDB installation, and then create the extension.
```

Comment:
follow the instructions described above --> follow the preceding instructions
(per Oracle style guide, no above-below for position within the document and no higher-lower for s/w versions)

Reply:
Dang it, I missed this one. Above/below is as bad as "see" for accessibility. Thanks!